src/main/kotlin/no/nav/syfo/consumer/dokarkiv/domain/DokarkivRequest.kt

```diff
@@ -27,6 +28,8 @@ data class DokarkivRequest(
             tema = "OPP", // Oppfolging
             kanal = "S",
             sak = Sak("GENERELL_SAK"),
+            // By default, user can not see documents created by others. Following enables viewing on Mine Saker:
+            overstyrInnsynsregler = "VISES_MASKINELT_GODKJENT",
```

Comment:
Stemmer det at vi skal ha default visning / alt skal bli synlig for bruker?

Reply:
Ja, det stemmer @AudunSorheim